src/client/Microsoft.Identity.Client/Internal/Broker/IBroker.cs

```diff
@@ -31,6 +31,7 @@ internal interface IBroker
             AuthenticationRequestParameters authenticationRequestParameters,
             AcquireTokenByUsernamePasswordParameters acquireTokenByUsernamePasswordParameters);
 
+        Dictionary<string, string> GetSsoPolicyHeaders();
```

Comment:
Would `DetermineSsoPolicyHeaders` be an appropriate name? given that the method's purpose is to resolve or determine which headers to use based on the SSO policy type Runtime returns?

Reply:
I would propose something like GetSsoPolicy, AcquireSsoPoilcy or along those line.

The determination is coming from windows api. But it shouldn't specific to the header since it could also be in cookie in some cases.

Reply:
interface is internal, don't dwell on this too long :)